src/Discord/Parts/Channel/Channel.php

```diff
@@ -702,6 +702,45 @@ function ($response) use ($deferred) {
         return $deferred->promise();
     }
 
+    /**
+     * Edit a message in the channel.
+     *
+     * @param Message $message  The message to edit.
+     * @param string  $text     The text to of the message.
+     * @param bool    $tts      Whether the message should be sent with text to speech enabled.
+     * @param Embed|array|null $embed An embed to send.
+     *
+     * @return PromiseInterface
+     * @throws \Exception
+     */
+    public function editMessage(Message $message, string $text, bool $tts = false, $embed = null): PromiseInterface
```

Comment:
To edit a message you are supposed to use `$channel->messages->save($message)` but we can leave this since it's easier.